cypress/support/fragments/inventory/inventoryInstance.js

```diff
@@ -1304,6 +1304,10 @@ export default {
     cy.expect(MultiColumnListCell({ content: itemStatus }).exists());
   },
 
+  clickHoldingAccordion: () => {
```

Comment:
We already have the function for opening holdings accordion InventoryInstance.openHoldingsAccordion();